ts/parts-more/RadialAxis.ts

```diff
@@ -746,17 +766,26 @@ radialAxisMixin = {
 
         // Spokes
         if (axis.isCircular) {
-            a = (typeof innerRadius === 'string') ?
-                relativeLength(innerRadius, 1) : (
-                    innerRadius /
-                    Math.sqrt(Math.pow(x2 - x1, 2) + Math.pow(y2 - y1, 2))
-                );
+            pyth =
```

Comment:
Took me a while :D Wouldn't be `distance` more descriptive name?

Reply:
Agree, the `pyth` is a little bit enigmatic :D.

Reply:
Changed to the `distance`.